### PR TITLE
pserialize_unregister: add ebr_unregister

### PR DESCRIPTION
**Pull request by outscale-gle**
fix a leak by changing pserialize_unregister and adding it to npf_destroy

### CODE REVIEW

File: src/kern/npf.c [hunk 1]
```diff
@@ -101,6 +101,7 @@ npf_destroy(npf_t *npf)
 	npf_conn_fini(npf);
 	npf_ifmap_fini(npf);
 
+	pserialize_unregister(npf->qsbr);
```
Comment (rmind) [inline src/kern/npf.c]: @outscale-gle: Why did you add this call here?

If the thread which performed `npf_create()` processes the packets, then it must register itself using `npf_thread_register()` and, symetrically, remove itself using `npf_thread_unregister()` before calling `npf_destroy()`.

**Comment by outscale-gle** [comment]
Ok i delete `pserialize_unregister` in` npf_destroy` and fix my leak in my program ! Sorry for this mistake and thanks for your time
